File: Hero/ClientApp/src/app/quote/models/subjectivity/subjectivity-configuration/SearchSubjectivitiesQuery.ts
import {RiskQuestionAnswer} from "@app/quote/models/subjectivity/subjectivity-configuration/RiskQuestionAnswer";

export class SearchSubjectivitiesQuery {
    public LanguageIsoCode: string;
    public HasLocalBroker: boolean;
    public InceptionDate: string;
    public ProductCode: string;
    public CountryIsoCode: string;
    public StateIsoCode: string;
    public BusinessLines: string[];
    public RiskQuestionAnswers: RiskQuestionAnswer[];
}

File: Hero/Models/SubjectivityConfiguration/SearchSubjectivitiesQuery.cs
namespace Hero.Models.SubjectivityConfiguration;

public class SearchSubjectivitiesQuery
{
    public string LanguageIsoCode { get; set; }
    public bool HasLocalBroker { get; set; }
    public string InceptionDate { get; set; }
    public string ProductCode { get; set; }
    public string CountryIsoCode { get; set; }
    public string StateIsoCode { get; set; }
    public string[] BusinessLines { get; set; }
    public RiskQuestionAnswer[] RiskQuestionAnswers { get; set; }
}